scripts/deconfigure.js

```diff
@@ -15,8 +15,8 @@ const config = packageJson.config
 module.exports = function() {
   modifyApigClient(config.apiGatewayApiId, config.primaryAwsRegion)
   modifyDevPortalAwsService(config.cognitoIdentityPoolId, config.primaryAwsRegion, config.cognitoRegion, config.cognitoUserPoolId, config.cognitoClientId)
-  modifySwaggerFile(config.accountId, config.primaryAwsRegion, config.apiGatewayApiName)
-  modifyExpressServer(config.siteS3Bucket, config.primaryAwsRegion, config.apiGatewayApiId)
+  // modifySwaggerFile(config.accountId, config.primaryAwsRegion, config.apiGatewayApiName)
+  // modifyExpressServer(config.siteS3Bucket, config.primaryAwsRegion, config.apiGatewayApiId)
   modifyPackageFile(config)
```

Comment:
Nice. Any reason to keep those functions or can we remove them completely?

Reply:
Nope. I'll remove them.